app-media-recorder.html

```diff
@@ -161,6 +161,17 @@
             type: Boolean,
             value: false,
             notify: true
+          },
+
+          /**
+           * When set to true, the recorder will only dispatch `app-media-recorder-chunk`
+           * events, and will not keep a local cache of the data chunks that have been
+           * recorded. A consequence of this is that a final `data` Blob will not be
+           * available when the recording has finished.
+           */
+          onlyChunks: {
```

Comment:
should this maybe be called "no-local-cache" or something? Disclaimer: I don't really know what a chunk or a blob is

Reply:
A "chunk" is a slice of recorded video. Typically, we receive them as a stream of events and collect them into an array (we also dispatch a `app-media-recorder-chunk` event for each one). They can't be used individually, but if you collect them in an array, you can make a `Blob` out of them. `Blob` is a global type in the browser that can be used as a container for binary data, and can also be used as source for things like the `<video>` and `<audio>` element.

So, you're right, this name doesn't really describe everything that is happening. I like `only` compared to `no`, because I like to name boolean properties after what they do rather than what they don't do.

What about like... `disableBlobCache`?

Reply:
Yes! I like that a lot

Reply:
Or, `disableBlogCreation`?